core/src/main/scala/org/apache/spark/rpc/RpcEndpointRef.scala

```diff
@@ -108,7 +108,7 @@ private[spark] abstract class RpcEndpointRef(conf: SparkConf)
 /**
  * An exception thrown if the RPC is aborted.
  */
-class RpcAbortException(message: String) extends Exception(message)
+private[spark] class RpcAbortException(message: String) extends Exception(message)
```

Comment:
shall we just make the `rpc` package private?

Reply:
Done in a42f54c.